doc/guide_3rd_party.md

```diff
@@ -142,16 +143,70 @@ Use a GSRN number from the response described in [Get Metering Points](#get-mete
 
 If the metering point owner does not already have a wallet, a new wallet will be created as part of the request. For both production and consumption certificates a contract is needed.
 
+```csharp
+var token = "<access-token>"; // Access token obtained with client-credentials
+var organizationId = "<organizationId>"; // One of the organizations you have consent to fetch data from
+var gsrn = "<gsrn>"; // One of the metering points GSRN's
+
+var contracts = new List<dynamic>();
+{
+    new
+    {
+        Gsrn = gsrn,
+        StartDate = DateTimeOffset.UtcNow.ToUnixTimeSeconds(),
+        EndDate = DateTimeOffset.UtcNow.AddDays(20).ToUnixTimeSeconds(),
+    }
+};
+var request = new { Contracts = contracts };
+var jsonString = JsonSerializer.Serialize(test);
+
+var httpClient = new HttpClient();
+var request = new HttpRequestMessage(HttpMethod.Post, "https://demo.energytrackandtrace.dk/api/certificates/contracts?organizationId=" + organizationId )
+{
+    Content = new StringContent(jsonString, Encoding.UTF8, "application/json")
+};
+request.Headers.Authorization = new AuthenticationHeaderValue("bearer", accessToken);
+request.Headers.Add("X-API-Version", "1");
+var response = await httpClient.SendAsync(request);
+```
+
```

Comment:
Der er også create wallet endpointet. skal det også dokumenteres?

Reply:
Nope. We don't want them to create a new wallet. We have talked about locking it down, since it can break alot of logic on our side if there is suddenly 2 wallets.